crates/bevy_ui/src/ui_node.rs

```diff
@@ -369,7 +370,7 @@ impl From<Color> for UiColor {
 }
 
 /// The image of the node
-#[derive(Component, Clone, Debug, Reflect)]
+#[derive(Component, Clone, Debug, Reflect, Deref)]
```

Comment:
We should have DerefMut to match IMO.

Reply:
```suggestion
#[derive(Component, Clone, Debug, Reflect, Deref, DerefMut)]
```

Reply:
We should have a full review of which things internally should use the `Deref` derives, I guess.